sale_automatic_workflow/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 
 {
     "name": "Sale Automatic Workflow",
-    "version": "14.0.1.4.0",
+    "version": "14.0.1.2.0",
```

Comment:
You should not modify this (if you don't provide migration scripts) as it will be updated by merge bot.